src/client/Microsoft.Identity.Client/AppConfig/ManagedIdentityIdType.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.AppConfig
+{
+    internal enum ManagedIdentityIdType
+    {
+        SystemAssigned,
```

Comment:
nit: Does 'SystemAssignedId' make sense here? or does SystemAssigned mean the system will chose between clientId or resourceId?

Reply:
SystemAssigned means system assigned identity, not user assigned ID. I see how some new to MSI may think abovel. I would assume people who work with MSI will understand the difference.

`SystemAssignedId` could work for consistency. 

We should make sure the enum/method comments are clear, though.

Reply:
Since there is no Id associated with System assigned managed identity, it is SystemAssigned. Client id and resource id are associated with user assigned managed identity.